iceoryx2-ffi/cxx/include/iox2/unique_port_id.hpp

```diff
@@ -36,9 +45,9 @@ class UniquePublisherId {
     void drop();
 
     iox2_unique_publisher_id_h m_handle = nullptr;
+    iox::optional<RawIdType> m_raw_id;
```

Comment:
Since the ID is const in Rust, it will be copied into the CXX layer on first access and the copy re-used for subsequent calls to `bytes()`.